wallet/src/wallet/changeset.rs

```diff
@@ -177,12 +188,26 @@ impl ChangeSet {
         )
     }
 
+    /// Get v1 sqlite [`ChangeSet`] schema. Schema v1 adds a table for locked outpoints.
+    pub fn schema_v1() -> alloc::string::String {
+        format!(
+            "CREATE TABLE {} ( \
+                txid TEXT NOT NULL, \
+                vout INTEGER NOT NULL, \
+                is_locked INTEGER, \
```

Comment:
I understand why `is_locked` is needed in the ChangeSet but I don't think we need an `is_locked` column. I'd expect that if we persist a ChangeSet with a `false` here we'd remove the record from the table rather than updated it with a `is_locked=false`. Otherwise this table would grow unbounded for un-locked locks that don't have an `expiration_date`.